db/migrations/setup_databases.sql

```diff
@@ -1,5 +1,2 @@
-CREATE DATABASE abodenb;
-CREATE TABLE listings(id SERIAL PRIMARY KEY, title VARCHAR(60));
 
-CREATE DATABASE abodenb_test;
-CREATE TABLE listings(id SERIAL PRIMARY KEY, title VARCHAR(60));
+CREATE TABLE listings(id SERIAL PRIMARY KEY, title VARCHAR(60), description VARCHAR(280), price INTEGER);
```

Comment:
if we use integer for price it means we cannot have decimal numbers as price. Not specified in user stories but maybe we should all agree on one?

Reply:
Yep true - the 'number' box I've got in the form only accepts whole numbers too, so floats can't currently be passed in as a price via the website. Up for discussion if we should change that (but as you say, not specified in the user stories ha)

Reply:
we will stick with integers